File: exercicios_py/secao6/exercicio4.py
#Tendo como entrada a altura e sexo, 
# construa um algoritmo que calcule seu peso
# ideal. Homens: (72.7*altura)-58; Mulheres 
# (62.1*altura) - 44.7

alt = float(input("Digite sua altura: "))
sx = str(input("Digite seu sexo (m/f): "))
if sx == "f":
    print("Seu peso ideal é de {}kg".format((62.1*alt) - 44.7))
elif sx == "m":
    print("Seu peso ideal é de {}kg".format((72.7*alt) - 58))
else:
    print("Sexo inválido")

File: exercicios_java/exercicios_java/src/secao6/exercicio4.java
//Tendo como entrada a altura e sexo, construa um algoritmo que calcule seu peso
//ideal. Homens: (72.7*altura)-58; Mulheres (62.1*altura) - 44.7
package secao6;
import java.util.Scanner;

public class exercicio4 {
    public static void main(String[] args){
        float alt, pid;
        char sx;
        Scanner t = new Scanner(System.in);

        System.out.print("Digite seu sexo (M/F): ");
        sx = t.next().charAt(0);
        System.out.print("Digite sua altura: ");
        alt = t.nextFloat();

        //diferença - '' char; "" string;

        if(sx == 'm'){
            pid = (float)(72.7*alt)-58;
            System.out.printf("Seu peso ideal é %.1f kg", pid);
        }else if(sx == 'f'){
            pid = (float)(62.1*alt-(float)44.7);
            System.out.printf("Seu peso ideal é %.1f kg", pid);
        }else{
            System.out.print("Sexo inválido");
        }

        t.close();
    }
}
